app/src/main/java/com/cornellappdev/uplift/ui/screens/ReportSubmittedScreen.kt

```diff
@@ -0,0 +1,146 @@
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_check_circle),
+                    contentDescription = "Check Circle",
+                )
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.spacedBy(10.dp)
+                ) {
+                    Text(
+                        text = "Thank you for your input!",
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Bold,
+                        fontSize = 16.sp,
+                    )
+                    Text(
+                        text = "Your report has been submitted.",
+                        fontFamily = montserratFamily,
+                        fontSize = 14.sp,
+                    )
+                }
+
+            }
+            Column(
+                verticalArrangement = Arrangement.spacedBy(16.dp),
+            ) {
+                Surface(
+                    color = GRAY01,
+                    shape = RoundedCornerShape(38.dp),
+                    modifier = Modifier
+                        .width(184.dp)
+                        .height(42.dp),
+                    shadowElevation = 2.dp,
+                    onClick = { onSubmitAnother() }
+                ) {
+                    Text(
+                        text = "SUBMIT ANOTHER",
+                        fontFamily = montserratFamily,
+                        fontSize = 14.sp,
+                        fontWeight = FontWeight.Bold,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.padding(12.dp)
+                    )
+                }
+                Row(
+                    modifier = Modifier
+                        .width(184.dp)
+                        .clickable(onClick = { onReturnHome() }),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(
+                        8.dp,
+                        Alignment.CenterHorizontally
+                    ),
+                ) {
+                    Text(
+                        text = "Return to Home",
+                        fontFamily = montserratFamily,
+                        fontSize = 14.sp,
+                        fontWeight = FontWeight.Medium,
+                        color = GRAY04
+                    )
+                    Icon(
+                        imageVector = Icons.AutoMirrored.Filled.ArrowForward,
+                        contentDescription = "Right Arrow",
+                        tint = GRAY05,
+                        modifier = Modifier
+                            .height(20.dp)
+
+                    )
+                }
+
+            }
+        }
+    }
+
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun ReportSubmittedScreenPreview() {
+    ReportSubmittedScreen()
+}
```

Comment:
I do want to make sure that this screen looks correct on several device heights. I actually can't really tell how you're centering the text from the composables above, but...

In general, suppose we centered an object on one device by padding some `300.dp` padding on top. If you then have a shorter screen, this might cause the object to render actually lower than the middle.

In general, usages of like `Modifier.weight(1f)` or `verticalArrangement` achieve a streamlined effect across screen sizes, but I wasn't quite sure if yours was doing anything like that. If you suspect that your code suffers from that problem above, plz fix!

Reply:
Hi, sorry for the disorganized composables, I extracted them into private composables, but the general idea for centering was I used 
```
verticalArrangement = Arrangement.spacedBy(
                60.dp,
                Alignment.CenterVertically
            ),
```
from the main outermost column to center the inner most composables vertically and then space them apart by 60.dp

Reply:
Ah okay I see, I actually didn't know you could used `spacedBy` WITH a center vertically... that's awesome!!